packages/expo-media-library/ios/EXMediaLibrary/EXMediaLibraryImageLoader.m

```diff
@@ -0,0 +1,106 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#if __has_include(<React/RCTBridge.h>)
+
+#import <Photos/Photos.h>
+#import <React/RCTUtils.h>
+#import <EXMediaLibrary/EXMediaLibraryImageLoader.h>
+
+@implementation EXMediaLibraryImageLoader
```

Comment:
How about some information about why this is needed in a comment above the class definition?